guides/doc-Administering_Red_Hat_Satellite/topics/adding_rhel_system_roles.adoc

```diff
@@ -1,16 +1,34 @@
 [[chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Managing_Ansible_Roles]]
 == Managing Ansible Roles
 
-In {Project}, you can import Ansible roles and Red{nbsp}Hat Enterprise Linux system roles to help with automation of routine tasks. Ansible is enabled by default on {Project} and {SmartProxy}.
+In {Project}, you can import Ansible roles to help with automation of routine tasks. Ansible is enabled by default only on {Project}.
 
-If you use custom or third party Ansible roles, you must add the roles to the `/etc/ansible/roles` directory of the {SmartProxy} or {Project} where you want to use them.
+[[chap-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Configuring_your_Deployment_to_Run_Ansible_Roles]]
+=== Configuring your Deployment to Run Ansible Roles
 
-You must import the Ansible roles into {ProjectServer} from the `/etc/ansible/roles` directory before you can use them.
+Complete this procedure to configure your {Project} deployment to run Ansible roles.
+
+.Procedure
+
+. Add the roles to the `/etc/ansible/roles` directory on the {Project} and all {SmartProxies} from where you want to use the roles. If you want to use custom or third party Ansible roles, ensure to configure an external version control system to synchronize roles between {Project} and {SmartProxies}.
+
+. On all {SmartProxies} that you want to use to run Ansible roles on hosts, enable the Ansible plug-in:
++
+----
+# satellite-installer --scenario capsule \
+--enable-foreman-proxy-plugin-ansible true
+----
+
+. Distribute SSH keys to enable {SmartProxies} to connect to hosts using SSH. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html-single/managing_hosts/index#sect-Managing_Hosts-Establishing_a_Secure_Connection_for_Remote_Commands[Distributing SSH Keys for Remote Execution] in the _Managing Hosts_ guide. {Project} runs Ansible roles the same way it runs remote execution jobs.
+
+. Import the Ansible roles into {Project}. For more information, see xref:sect-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Importing_Ansible_Roles[].
+
+. Proceed to https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html-single/managing_hosts/index#Using_Ansible_Roles[Using Ansible Roles] in the _Managing Hosts_ guide.
 
 [[sect-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Importing_Ansible_Roles]]
 === Importing Ansible Roles
 
-You can import Ansible roles from a {SmartProxy} that has Ansible enabled or from the `/etc/ansible/roles` directory where {ProjectServer} is installed.
+You can import Ansible roles from the `/etc/ansible/roles` directory on {Project} or on a {SmartProxy} that has Ansible enabled. Ensure that roles that you import are located in the `/etc/ansible/roles` directory on all {SmartProxies} from where you want to use the roles.
```

Comment:
"Ensure that the roles you import are located in.. "

Reply:
Done.

Reply:
Still needs a minor edit: "Ensure that the roles ~that~ you import are located in the.."

Reply:
This "that" actually adds clarity and is especially useful for machine translation. It is a recommended practice in tech writing to use this "that" where possible.

Reply:
OK, works for me.